src/win/core.c

```diff
@@ -555,6 +565,10 @@ static void uv__poll(uv_loop_t* loop, DWORD timeout) {
          * meant only to wake us up.
          */
         if (overlappeds[i].lpOverlapped) {
+          uv__metrics_inc_events(loop, count);
```

Comment:
Not sure but shouldn't it be increased only by 1?
```suggestion
          uv__metrics_inc_events(loop, 1);
```

Reply:
yes